src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java

```diff
@@ -63,6 +68,86 @@ public void sendEmail(CaseDetails caseDetails) {
 
     }
 
+    public void sendEmailToApplicantAndRespondent(CaseDetails caseDetails) {
+        log.info("Sending the new manage order emails for Applicant/Respondent {}",caseDetails.getId());
+        CaseData caseData = emailService.getCaseData(caseDetails);
+        SelectTypeOfOrderEnum isFinalOrder = caseData.getSelectTypeOfOrder();
+        if (caseData.getCaseTypeOfApplication().equalsIgnoreCase("C100")) {
+            Map<String, String> applicantsMap = getEmailPartyWithName(caseData
+                                                                         .getApplicants());
+            Map<String, String> respondentMap = getEmailPartyWithName(caseData
+                                                                         .getRespondents());
+            for (Map.Entry<String, String> appValues : applicantsMap.entrySet()) {
+
+                emailService.send(
+                    appValues.getKey(),
+                    (isFinalOrder == SelectTypeOfOrderEnum.finl) ? EmailTemplateNames.CA_DA_FINAL_ORDER_EMAIL
+                        : EmailTemplateNames.CA_DA_MANAGE_ORDER_EMAIL,
+                    buildApplicantRespondentEmail(caseDetails, appValues.getValue()),
+                    LanguagePreference.english
+                );
+            }
+
+            for (Map.Entry<String, String> appValues : respondentMap.entrySet()) {
+
+                emailService.send(
+                    appValues.getKey(),
+                    (isFinalOrder == SelectTypeOfOrderEnum.finl) ? EmailTemplateNames.CA_DA_FINAL_ORDER_EMAIL
+                        : EmailTemplateNames.CA_DA_MANAGE_ORDER_EMAIL,
+                    buildApplicantRespondentEmail(caseDetails, appValues.getValue()),
+                    LanguagePreference.english
+                );
+            }
+        } else {
+            if (caseData.getApplicantsFL401().getEmail() != null) {
+                emailService.send(
+                    caseData.getApplicantsFL401().getEmail(),
+                    (isFinalOrder == SelectTypeOfOrderEnum.finl) ? EmailTemplateNames.CA_DA_FINAL_ORDER_EMAIL :
+                        EmailTemplateNames.CA_DA_MANAGE_ORDER_EMAIL,
+                    buildApplicantRespondentEmail(caseDetails, caseData.getApplicantsFL401().getFirstName()
+                        + " " + caseData.getApplicantsFL401().getFirstName()),
+                    LanguagePreference.english
+                );
```

Comment:
This can also be moved to a single method and same method can be used for code between line#113-120

Reply:
created common method to address this